### DISCUSSION TITLE
Change default scope for all requests

### DISCUSSION THREAD

**Discussion by vmsh0** [Byron/google-apis-rs#325] [Q&A] [answered]
Hello,

Is it possible to specify a default scope for all requests?

I would like to use the `/drive` scope for all requests, as it is the common denominator of all requests my application performs. Even when a `/drive` token is avaible in the authenticator, requests which have a narrower scopes as their default scope will request a new token. This forces the user to perform authorization multiple times.

I could, of course, manually specify the `/drive` scope for each single request. But I'm wondering whether there is a canonical way to force a scope for all requests - or whether there should be.

**Comment by Byron** [comment, marked as answer]
At the moment, there is no way I know of.

It should be possible to have a feature for that, and it would be appreciated, too, as this appears to be a desirable capability.
